### DISCUSSION TITLE
My Bright Lil' Mona

### DISCUSSION THREAD

**Discussion by lizsal** [martinwoodward/octolamp#31] [Show and tell]
This was an easy project - I loved it! And all the WLED presets are so much fun to check out during meetings. Wait, I mean .... 

Thank you, Martin!!

<img src="https://github.com/martinwoodward/octolamp/assets/43018285/6e73d6e7-2e9a-4bca-a6dd-931505b807ca" width="250">

<img src="https://github.com/martinwoodward/octolamp/assets/43018285/6ab8c890-3f5d-463c-92c6-134c9268aa83)" width="250">

**Comment by martinwoodward** [comment]
Love your Tidbyt in the background also :) - Did you go with white or translucent filament?  Looks slightly more translucent than the ones I've done but might also be the lighting.

**Comment by lizsal** [comment]
I used [silk white](https://www.amazon.com/gp/product/B07HXJPC64/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1). And [silk coffee gold](https://www.amazon.com/gp/product/B07V28MLC8/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1) on the outside. Goes with my office 😁

**Reply by martinwoodward** [reply]
_Fancy_.  Looks awesome.

**Reply by lizsal** [reply]
The pic with the lights on is on the WLED "washing machine" config with brightness cranked and who knows what photo settings that pic up each individual light on well. The light actually diffuses and blends nicely, which a gif or video would show better.
